DEVELOPMENT.md

```diff
@@ -224,10 +223,6 @@ Upon opening, VSCode may prompt you to install recommended extensions. Go ahead
 
```

Comment:
It won't let me attach this to the above line, but I don't think the 'Go ahead and accept installing the recommended extensions' applies here anymore - I see that this is the README for devs, but Prettier is a different "type" of recommended extension than the other two. Instead, could we add a very short description alongside each recommended extension for when these would be needed?

eg:
Prettier - Code formatter needed to pass CI formatting checks for PRs
Jar Viewer - DFDL schema files can be packed in jar files when passed to Daffodil. This extension allows the debugger to show these schema files while stepping through them.
Highlight Matching Tag - Convenience extension to better highlight opening and closing tags in DFDL schema files